pvlib/pvsystem.py

```diff
@@ -852,6 +852,67 @@ def i_from_v(self, resistance_shunt, resistance_series, nNsVth, voltage,
         return i_from_v(resistance_shunt, resistance_series, nNsVth, voltage,
                         saturation_current, photocurrent)
 
+    def get_ac(self, model, p_dc, v_dc=None):
```

Comment:
I believe this needs `@_unwrap_single_value` but I could be wrong.

Reply:
I think it should. Now I'm wondering why the tests didn't fail.

Reply:
`@_unwrap_single_value` shouldn't be used for the `get_ac` method, since every inverter function returns a numeric. There are no "per-array" outputs.

Reply:
ah, of course. the docstring incorrectly says `power_ac : numeric, or tuple of numeric`

Reply:
Corrected now.